packages/apps/pagarme/tsconfig.json

```
@@ -0,0 +1,3 @@
+{
+  "extends": "../../../tsconfig.json",
+}
```

```suggestion
{
  "extends": "../../../tsconfig.json",
  "compilerOptions": {
    "declaration": true
  }
}
```

App com módulos exporta uma função chamada em código, então é bom ter type declarations...